### PR TITLE
Add support for multiple PID/TID for offwaketime

### PR DESCRIPTION
**Pull request by lorddoskias**
Here's multi PID/TID support for offwaketime tool.

### CODE REVIEW

**Comment by yonghong-song** [comment]
[buildbot, ok to test]

File: tools/offwaketime.py [hunk 1]
```diff
@@ -20,6 +20,16 @@
 
 # arg validation
 def positive_int(val):
+    dest = []
```
Comment (yonghong-song) [inline tools/offwaketime.py:23]: The change of this function from returning a int to a list caused the following error
```
  File "../../tools/offwaketime.py", line 125, in <module>
30:     duration = int(args.duration)
30: TypeError: int() argument must be a string or a number, not 'list'
```

Comment (lorddoskias) [reply tools/offwaketime.py:23]: Yes the fix is to use _positive_int for the type of duration. I did catch it and updated my branch. I see that the commit has this updated.

**Comment by yonghong-song** [comment]
[buildbot, test this please]

File: tools/offwaketime.py (hunk 1 above)
Comment (yonghong-song) [reply tools/offwaketime.py:23]: > Yes the fix is to use _positive_int for the type of duration. I did catch it and updated my branch. I see that the commit has this updated.

The error still there. Could you double check?

Comment (lorddoskias) [reply tools/offwaketime.py:23]: Ops, I had missed the usage of positive_int in positive_nonzero_int. Pushed fixed branch now. Restart the tests.

**Comment by yonghong-song** [comment]
LGTM. Thanks!

**Pull request merged**